cvat/apps/documentation/user_guide.md

```diff
@@ -147,6 +147,8 @@ Go to the [Django administration panel](http://localhost:8080/admin). There you
       ![](static/documentation/images/image128.jpg)  
 
     **Z-Order**. Defines the order on drawn polygons. Check the box for enable layered displaying.
+
+    **Use zip chunks**. Force to use zip chunks as compressed data. Actual for videos only.
```

Comment:
@azhavoro , could you please work with Timur to resolve the unclear description? We wanted to update it in the UI as well.